docs/UserGuide.md

```diff
@@ -41,146 +37,167 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
-### 2.1. Viewing help : `help`
+### Viewing help : `help`
 
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### Adding a task: `addTask`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the task manager<br>
+Format: `addTask TASKNAME d/DEADLINE DATE [t/DEADLINE TIME] [p/PRIORITY] [#TAG]...`
 
-> Persons can have any number of tags (including 0)
+> Tasks can have any number of tags (including 0)
+> Tasks can only have one priority
+> Tasks can only have one deadline time
+> Tasks must have a deadline date
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `addTask completeUserGuide d/28-Feb-2017 t/2400 p/high #Important`
 
-### 2.3. Listing all persons : `list`
+### Adding a floating task: `addFTask`
+
+Adds a floating task to the task manager<br>
+Format: `addFTask FLOATINGTASKNAME [p/PRIORITY] [#TAG]...`
+
+> Floating Tasks can have any number of tags (including 0)
+> Floating Tasks can only have one priority
+> Floating Tasks cannot have any deadlines 
+
+Examples:
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+* `addFTask reachDiamondRank p/low #Overwatch`
```

Comment:
You can consider an alternative of using a single add command to create different types of tasks.